### ISSUE TITLE
Tables: right alignment for financial tables

### ISSUE THREAD

**Issue by sviens** [wet-boew/wet-boew#8156]
WET 3.1 had a great feature that enabled quick coding of right-aligned financial data in a financial table (screenshots below). I would like to request this feature be reinstated in WET 4 as soon as possible.

Thank you!

![capture1](https://user-images.githubusercontent.com/2761556/30866809-40a2b2b8-a2a8-11e7-8a5f-17f9118703b8.JPG)
![capture2](https://user-images.githubusercontent.com/2761556/30866819-44e6415a-a2a8-11e7-99b6-b736a1d8e13e.JPG)

**Comment by Bgolden67** [comment]
instead of align-right use text-right

**Comment by DavidCrepeault** [comment]
Isn't text-right applied to the cell?  We need a solution that applies to entire financial tables.

**Comment by RobJohnston** [comment]
What @Bgolden67 said will work for the `<td>` cells, but the `<th>` cells will still be left-aligned.

**Comment by Bgolden67** [comment]
@RobJohnston correct, but  means a lot less coding to apply text-center to the th cells if you need them centered across the top, and works fine for 1st column row headers
